python/packages/autogen-core/src/autogen_core/model_context/_unbounded_buffered_chat_completion_context.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, List, Mapping
+
+from ..models import LLMMessage
+from ._chat_completion_context import ChatCompletionContext
+
+
+class UnboundedBufferedChatCompletionContext(ChatCompletionContext):
```

Comment:
`UnboundedChatCompletionContext` ? Since it is already unbounded there is no buffer.